routers/init.go

```diff
@@ -185,9 +185,15 @@ func NormalRoutes(ctx context.Context) *web.Route {
 	r.Mount("/", web_routers.Routes(ctx))
 	r.Mount("/api/v1", apiv1.Routes(ctx))
 	r.Mount("/api/internal", private.Routes())
+
 	if setting.Packages.Enabled {
-		r.Mount("/api/packages", packages_router.Routes(ctx))
-		r.Mount("/v2", packages_router.ContainerRoutes(ctx))
+		// Add endpoints to match common package manager APIs
+
+		// This implements package support for most package managers
+		r.Mount("/api/packages", packages_router.CommonRoutes(ctx))
+
+		// This implements Docker HUB API (Note this is not preceded by /api but is instead /v2)
+		r.Mount("/v2", packages_router.DockerContainerRoutes(ctx))
```

Comment:
It's OCI compatible implementation not only Docker, so I think keeping the original name is better.

Reply:
OCIContainerRoutes?

Reply:
It maybe a better name but since this PR's emphasis is to change the documentation, I don't think this should block the PR.

Reply:
`OCIContainerRoutes` = `OpenContainerInitiativeContainerRoutes`
I would just keep `ContainerRoutes`. The comments tell you that these are for the OCI API.